### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by BlackBeltPanda** [Falseee/Builder-s-Wand#13]
Getting spammed with this error with Builder's Wand 1.4.0 in MC 1.12: https://pastebin.com/raw/b5vywZLz

**Comment by Falseee** [comment]
Please show me your config.

**Comment by BlackBeltPanda** [comment]
Config: https://pastebin.com/raw/gcnxwYaR

**Comment by Falseee** [comment]
Do you get the error instantly after the plugin is loaded ? (or do you have to rightclick with the wand or anything else?).
Im correct that you use also worldguard ? (can you test it with the newest worldguard version ?).

I cant replication it, so i need your help to fix it :)

**Comment by BlackBeltPanda** [comment]
It seems to happen when players use the wand (right click).
I'm using WorldGuard 6.2.1

**Comment by BlackBeltPanda** [comment]
I also just noticed this error in the logs:
```
[Server thread/WARN]: [BuildersWand] Task #572 for BuildersWand v1.4.0 generated an exception
java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:653) ~[?:1.8.0_131]
	at java.util.ArrayList.get(ArrayList.java:429) ~[?:1.8.0_131]
	at de.False.BuildersWand.items.Wand$1.run(Wand.java:94) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:58) ~[patched_1.12.jar:git-Paper-1143]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:356) [patched_1.12.jar:git-Paper-1143]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:832) [patched_1.12.jar:git-Paper-1143]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) [patched_1.12.jar:git-Paper-1143]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:766) [patched_1.12.jar:git-Paper-1143]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:664) [patched_1.12.jar:git-Paper-1143]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
```

**Comment by Falseee** [comment]
I have the same WorldGuard Version and i get no errors...
Can you test if you also get the error when you only have worldguard and builders-wand installed ? (just create a temp server with the same versions).

I will check the second (new) error.

**Comment by BlackBeltPanda** [comment]
I'm having a tough time replicating it but it shows up often when my players use it. I'll have to see if it happens in the new version.

**Comment by Falseee** [comment]
@BlackBeltPanda 
Okay, i will add a internal storage/inventory for the builders wand then its ready for the release today ! :)

**Comment by Falseee** [comment]
@BlackBeltPanda 
Maybe fixed in the new Version (v2.2.0), can you test if you still get the errors ?

**Comment by Falseee** [comment]
If you have this issue again pls pm me or open a new issue :)
